src/Service/ExcelExport.php

```diff
@@ -227,14 +241,21 @@ protected function addItemRow(Worksheet $sheet, int $y, array $row): void
         }
 
         if (!empty($row['extra']['customFields'])) {
-            $customFields = $row['extra']['customFields'];
-            $columns = ['M', 'N', 'O'];
-            $c = 0;
-
-            foreach ($customFields as $field => $value) {
-                $sheet->setCellValue("$columns[$c]1", $field);
-                $sheet->setCellValue("$columns[$c]$y", $value);
-                ++$c;
+            $customFieldsData = $row['extra']['customFields'];
+
+            foreach ($customFieldsData as $field => $value) {
```

Comment:
It would probably be better to loop over the columns 13 through 12+count(static::$customFields), and then get the value for the field for that column from the item and set it (i.e. just one loop instead of having an inner loop).

Reply:
Refactored code pushed for your review.